sdcm/tester.py

```diff
@@ -386,15 +386,27 @@ def get_cluster_gce(self, loader_info, db_info, monitor_info):
             monitor_info['n_local_ssd'] = self.params.get('gce_n_local_ssd_disk_monitor')
 
         user_prefix = self.params.get('user_prefix', None)
-        gce_datacenter = self.params.get('gce_datacenter', None).split()
+        regions = self.params.get('gce_datacenter', None).split()
         service_cls = get_driver(Provider.GCE)
         key_store = KeyStore()
         gcp_credentials = key_store.get_gcp_credentials()
         services = []
-        for i in gce_datacenter:
+        gce_datacenter = []
+        for region_az in regions:
+            # dc can be "us-east1-b" or "us-east1"
+            assert "us-east1" in region_az, "only 'us-east1' region is supported"
+            if region_az.count("-") == 1:
+                # randomly select zone and add it as a
+                # https://cloud.google.com/compute/docs/regions-zones#available
+                zone = random.choice(["c", "d"])
```

Comment:
What happened to a and b?

Reply:
there is no "a" zone in us-east1, only b,c,d. Check here: https://cloud.google.com/compute/docs/regions-zones#locations